### DISCUSSION TITLE
Project directory . is not empty

### DISCUSSION THREAD

**Discussion by adam314315** [symfony/symfony#57505] [Q&A]
### Symfony version(s) affected

7.1.*

### Description

Hi,

After cloning an existing git repo to install a new fresh Symfony I got the error Project directory not empty due to the existing .git folder I suspect. What is the recommended best practices/steps starting with an existing repo to install a symfony version ? 

Steps to reproduce the error :
git clone myrepo
cd myrepo
symfony new .  --webapp

>> Error 
Project directory . is not empty

### How to reproduce

Steps to reproduce the error :
git clone myrepo
cd myrepo
symfony new .  --webapp

**Comment by MatTheCat** [comment]
Hi, this has already been discussed in https://github.com/composer/composer/issues/7455. You’ll find explanations and a work-around there.

**Reply by sovetski** [reply]
Nothing found about this problem

**Comment by xabbuh** [comment]
Use another directory than `.` when running `symfony new` and afterwards move the files into the root of your git repository.

**Reply by adam314315** [reply]
Yes this is what I am currently doing but looks more a workaround than a perfect solution for me.

**Reply by xabbuh** [reply]
You can open an issue in the [repository of the CLI](https://github.com/symfony-cli/symfony-cli) to request this feature to see if there is a chance to get this feature implemented.

**Reply by adam314315** [reply]
Ok good idea. I will give it a try.
